tests/integration2/galley/conversion/testdata/load.go

```diff
@@ -50,14 +75,71 @@ func Load() ([]TestInfo, error) {
 			meshConfigFile = ""
 		}
 
-		r := TestInfo{
+		fs := &FileSet{
 			inputFile:      asset,
 			expectedFile:   expectedFile,
 			meshConfigFile: meshConfigFile,
-			Skipped:        false,
 		}
-		result = append(result, r)
+
+		if len(info.files) <= index {
+			arr := make([]*FileSet, index+1)
+			copy(arr, info.files)
+			info.files = arr
+		}
+
+		if info.files[index] != nil {
+			return nil, fmt.Errorf("Duplicate entry found for %q at index %d", baseFileName, index)
+		}
+		info.files[index] = fs
 	}
 
+	var result []*TestInfo
+
+	// Do a sanity check to ensure there is data for each stage. Also order the info, so that things can run
```

Comment:
nit: consider splitting this comment so that the info regarding sorting is by the sorting code.